massifs/watcher/watcher.go

```diff
@@ -62,16 +62,17 @@ func (w Watcher) ConfigString() string {
 }
 
 func ConfigDefaults(cfg *WatchConfig) error {
-	if cfg.Since.UnixMilli() == 0 || cfg.IDSince != "" && cfg.Horizon != 0 {
+	zeroTime := time.Time{}
+	if cfg.Since.UnixMilli() == zeroTime.UnixMilli() && cfg.IDSince == "" && cfg.Horizon == 0 {
```

Comment:
Apparently comparing times should use the builtin methods viz.

       if cfg.Since.Equal(zeroTime)...

although I do not see a problem with the above code if you want to preserve millisecond accuracy...

Reply:
yeah, and that results in clearer code anyway.